src/workerd/jsg/modules.h

```diff
@@ -267,15 +354,40 @@ class ModuleRegistryImpl final: public ModuleRegistry {
   // we need to be able to search it by path (filename) as well as search for a specific module
   // object by identity. We use a kj::Table!
   struct Entry {
+    using Info = kj::OneOf<ModuleInfo,
+                           kj::ArrayPtr<const char>,
+                           kj::Function<ModuleInfo(Lock&)>>;
+
+    struct Key {
+      const kj::Path& specifier;
+      Type type = Type::BUNDLE;
```

Comment:
Should this be const? I assume that we are not expecting the type to change.

Reply:
It certainly can be, yes.